internal/core/workers/operation_execution_worker/operation_execution_worker.go

```diff
@@ -152,7 +152,7 @@ func (w *OperationExecutionWorker) Start(ctx context.Context) error {
 
 		executeStartTime := time.Now()
 		executionErr := executor.Execute(operationContext, op, def)
-		reportOperationExecutionLatency(executeStartTime, w.schedulerName, op.DefinitionName, executionErr != nil)
+		reportOperationExecutionLatency(executeStartTime, w.schedulerName, op.DefinitionName, executionErr == nil)
```

Comment:
Hey @guilhermocc what do you think to use the same approach from schedulers' adapters? On that, it would receive a function in the `reportOperationExecutionLatency` instead of the start time.